api/tests/prescription/learner-management/unit/application/api/learners-api_test.js

```diff
@@ -29,4 +32,58 @@ describe('Unit | Prescription | learner management | Api | learners', function (
       expect(hasBeenLearnerStub).to.have.been.calledOnceWith({ userId });
     });
   });
+
+  describe('#deleteOrganizationLearnerBeforeImportFeature', function () {
+    let findOrganizationLearnersBeforeImportFeatureStub, deleteOrganizationLearnersStub;
+
+    beforeEach(function () {
+      findOrganizationLearnersBeforeImportFeatureStub = sinon
+        .stub(usecases, 'findOrganizationLearnersBeforeImportFeature')
+        .rejects();
+      deleteOrganizationLearnersStub = sinon.stub(usecases, 'deleteOrganizationLearners').rejects();
+    });
+
+    it('should throw a "TypeError" when "userId" is not defined', async function () {
+      // when
+      const error = await catchErr(deleteOrganizationLearnerBeforeImportFeature)({
+        userId: undefined,
+        organizationId: Symbol('organizationId'),
+      });
+
+      // then
+      expect(error).to.be.instanceOf(TypeError);
+      expect(findOrganizationLearnersBeforeImportFeatureStub).to.not.have.been.called;
+      expect(deleteOrganizationLearnersStub).to.not.have.been.called;
+    });
+
+    it('should throw a "TypeError" when "orgnaizationId" is not defined', async function () {
+      // when
+      const error = await catchErr(deleteOrganizationLearnerBeforeImportFeature)({
+        userId: Symbol('userId'),
+        organizationId: null,
```

Comment:
**nitpick**: plutôt mettre organizationId: undefined pour être iso avec le précédent ?

Typo dans le it 😆  'orgnaizationId'